go/vt/vtgate/vindexes/binary_test.go

```diff
@@ -24,25 +24,72 @@ import (
 	"reflect"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"vitess.io/vitess/go/sqltypes"
 	"vitess.io/vitess/go/vt/key"
+	"vitess.io/vitess/go/vt/proto/vtrpc"
+	"vitess.io/vitess/go/vt/vterrors"
 )
 
 var binOnlyVindex SingleColumn
 
 func init() {
-	vindex, _ := CreateVindex("binary", "binary_varchar", nil)
+	vindex, warnings, err := CreateVindex("binary", "binary_varchar", nil)
```

Comment:
Adding warnings validations anywhere `CreateVindex` is used in tests in the `vindexes` package.